README.md

```diff
@@ -303,4 +303,4 @@ Got questions? Feel free to ask at the [RedisGraph forum](https://forum.redislab
 
 ## License
 
-Redis Source Available License Agreement - see [LICENSE](LICENSE)
+Licensed under your choice of the Redis Source Available License 2.0 (RSALv2) or the Server Side Public License v1 (SSPLv1). See LICENSE.
```

Comment:
Licensed under your choice of the Redis Source Available License 2.0 (RSALv2) or the Server Side Public License v1 (SSPLv1). See [LICENSE](LICENSE.txt).